impeller/renderer/backend/vulkan/render_pass_builder_vk.cc

```diff
@@ -111,6 +127,22 @@ vk::UniqueRenderPass RenderPassBuilderVK::Build(
                                                     kUnusedAttachmentReference);
   vk::AttachmentReference depth_stencil_ref = kUnusedAttachmentReference;
 
+  if (color0_.has_value()) {
+    vk::AttachmentReference color_ref;
+    color_ref.attachment = attachments.size();
+    color_ref.layout = vk::ImageLayout::eGeneral;
+    color_refs[0] = color_ref;
+    attachments.push_back(color0_.value());
+
+    if (color0_resolve_.has_value()) {
+      vk::AttachmentReference resolve_ref;
+      resolve_ref.attachment = attachments.size();
+      resolve_ref.layout = vk::ImageLayout::eGeneral;
+      resolve_refs[0] = resolve_ref;
+      attachments.push_back(color0_resolve_.value());
+    }
```

Comment:
Please remove duplication

Reply:
Done

Reply:
Can't you use the Iterate method here to avoid the duplication?

Reply:
This isn't the render_target, this is the internal implementation of the RenderPassBuilderVK